preprocessors/semanticSeg/segment.py

```diff
@@ -106,12 +110,14 @@ def findContour(pred_color, width, height):
             contour_indi[j][1] = float(float(contour_indi[j][1]) / height)
```

Comment:
likewise, would log the number of expected iterations of this loop before entering